src/NServiceBus.Msmq.AcceptanceTests/When_TimeToBeReceived_set_and_native_receivetransaction.cs

```diff
@@ -43,6 +45,7 @@ public class MyMessageHandler : IHandleMessages<MyMessage>
 
                 public async Task Handle(MyMessage message, IMessageHandlerContext context)
                 {
+                    Context.HandlerInvoked = true;
```

Comment:
since we're completing all inflight messages when before the endpoint shuts down, we should be good.